Microsoft.SystemForCrossDomainIdentityManagement/Protocol/PatchOperation2Combined.cs

```diff
@@ -32,6 +33,10 @@ public string Value
         {
             get
             {
+                if(this.values == null)
+                {
+                    this.values = this.Path.SubAttributes.First()?.ComparisonValue;
```

Comment:
this.Path could also be null. A better approach to do this would be during deserialization. You should parse the web-request while de-serializing the object into a PATCH object. That would be a place where you could de-serialize it correctly and do other validations on it.